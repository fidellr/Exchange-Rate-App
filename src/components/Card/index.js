import React from 'react';
import cls from 'classnames';

export default ({
  className,
  children,
}) => {
  const classes = cls('u-boxShadowBottom u-borderRadius2', className);

  return (
    <div className={classes}>
      {children}
    </div>
  )
};
