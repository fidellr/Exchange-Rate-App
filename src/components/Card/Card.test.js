import React from 'react';
import { shallow } from 'enzyme';
import Card from './index';

describe('Card', () => {
  it('should render with default className as it specified', () => {
    const component = shallow(<Card className="cardTest-clsName" />);
    expect(component.hasClass('u-boxShadowBottom u-borderRadius2 cardTest-clsName')).toBeTruthy();
    expect(component).toMatchSnapshot();
  });
});

