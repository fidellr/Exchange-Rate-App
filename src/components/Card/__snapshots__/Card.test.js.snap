// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render with default className as it specified 1`] = `
<div
  className="u-boxShadowBottom u-borderRadius2 cardTest-clsName"
/>
`;
